app/views/layouts/admin.html.erb

```diff
@@ -84,6 +84,12 @@
         </div>
       <% end %>
 
+      <div
```

Comment:
This code sets off a listener for any `generateErrorAlert` events, and responds appropriately.